test/e2e/storage/testsuites/volumes.go

```diff
@@ -178,9 +178,9 @@ func (t *volumesTestSuite) defineTests(driver TestDriver, pattern testpatterns.T
 		// local), plugin skips setting fsGroup if volume is already mounted
 		// and we don't have reliable way to detect volumes are unmounted or
 		// not before starting the second pod.
-		volume.InjectContent(f.ClientSet, config, fsGroup, pattern.FsType, tests)
+		volume.InjectContent(f, f.ClientSet, config, fsGroup, pattern.FsType, tests)
```

Comment:
Looks like we still have a redundant parameter here: `f.ClientSet` can be removed.

Reply:
removed clientset parameter, rebased and force-pushed.